### ISSUE TITLE
graphviz seems to need the url update

### ISSUE THREAD

**Issue by hacker** [Homebrew/brew#1829]
When trying to upgrade graphviz, I encountered this:

> ==> Downloading http://graphviz.org/pub/graphviz/stable/SOURCES/graphviz-2.40.1.tar.gz
> 
> curl: (6) Could not resolve host: graphviz.org
> Error: Failed to download resource "graphviz"

Indeed, graphviz.org doesn't resolve, unlike www.graphviz.org, which does. And serves the file in question too.

**Comment by apjanke** [comment]
I'm unable to reproduce. The download works for me (in Brooklyn), both in `brew` and `wget`.

May be a transient or local DNS issue?

**Comment by hacker** [comment]
Sorry, yes, it resolves now. And no, it wasn't local DNS, I tried with many nameservers, including google ones, and even from different locations before I tried www. I ended up adding hosts entry with the same ip as www (because that was easier than finding out where to put the file and cleaner than editing the url), so I had to resolve it this way or another… So it must have been graphviz's DNS issue.

Anyway, sorry again for the false alert, I'm closing it now.
